pkgs/applications/window-managers/river/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, stdenv ,fetchFromGitHub
+, zig, pkg-config, scdoc
+, wayland ,xwayland, wayland-protocols, wlroots
+, libxkbcommon, pixman, udev, libevdev, libX11, libGL
+}:
+
+stdenv.mkDerivation rec {
+  pname = "river";
+  version = "unstable-2021-04-08";
+
+  src = fetchFromGitHub {
+    owner = "ifreund";
+    repo = "river";
+    rev = "9e3e92050e04320949c6cd995273c30319ebd515";
+    sha256 = "1v8dpbadsb3c7bc84sai09dbqv5s5s5d77vs12kdkd45x0ppmk3j";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = [ wayland xwayland wayland-protocols wlroots pixman
```

Comment:
Is a binary from wayland executed at build time? If so it needs to be in nativeBuildInouts.

Reply:
Wayland is xml file, there is no binary for it, I think it's runtime dependencie